SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,46 @@
+header:
+  schema-version: '1.0.0'
+  expiration-date: '2025-01-23T01:00:00.000Z'
+  last-updated: '2024-01-23'
+  last-reviewed: '2024-01-23'
+  project-url: https://github.com/kumahq/kuma
+  changelog: https://github.com/kumahq/kuma/blob/master/CHANGELOG.md
+  license: https://github.com/kumahq/kuma/blob/master/LICENSE
+project-lifecycle:
+  stage: active
+  bug-fixes-only: false
+  core-maintainers:
+    - github:bartsmykla
+    - github:lobkovilya
+    - github:lukidzi
+    - github:Automaat
+    - github:slonka
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+  code-of-conduct: https://github.com/kumahq/kuma/blob/master/CODE_OF_CONDUCT.md
+distribution-points:
+  - https://github.com/kumahq/kuma
+documentation:
+  - https://kuma.io/docs/
+security-contacts:
+  - type: email
+    value: maintainer@example.com
```

Comment:
@lahabana , is there a security@kuma.io or something else or is this nominative ?

Reply:
An URL can also be used